docs/generate.php

```diff
@@ -104,11 +86,19 @@ class BuilderParser
         'getSubscribedServices',
     ];
 
-    private string $name;
+    private const EXAMPLE_TEMPLATE = '
+```php
+return $gotenberg
+    // Your builder call as ->html() and the rest of your configuration code
+    {% method %}
```

Comment:
```suggestion
    ->{% method %}
```

to define an example without prefixing it with `->` . Simpler IMO.